src/components/Menu.scss

```diff
@@ -0,0 +1,73 @@
+.Menu{
```

Comment:
There are different ways to structure a project. When it comes to  `scss` files, I would recommend either having them all in a single folder or create a folder per component with its tsx and scss. That is:

Option 1: stylings in same folder
```
/src/styles/components/menu.scss
```

Option 2: Components in folder
```
/src/components/Menu/
                                        ---> Menu.tsx
                                        ---> Menu.scss
```

I'd recommend option 1

Reply:
I think all "style files" in style folder will be better to manage and modify